e2e/errors.go

```diff
@@ -43,3 +43,65 @@ func isRetryableAPIError(err error) bool {
 
 	return false
 }
+
+//nolint:lll // sample output cannot be split into multiple lines.
+/*
+getStdErr will extract the stderror and returns the actual error message
+
+Sample kubectl output:
+
+error running /usr/local/bin/kubectl --server=https://192.168.39.67:8443 --kubeconfig=***** --namespace=default create -f -:
+Command stdout:
+
+stderr:
+Error from server (AlreadyExists): error when creating "STDIN": services "csi-rbdplugin-provisioner" already exists
+Error from server (AlreadyExists): error when creating "STDIN": deployments.apps "csi-rbdplugin-provisioner" already exists
+
+error:
+exit status 1
+
+Sample message returned from this function:
+
+Error from server (AlreadyExists): error when creating "STDIN": services "csi-rbdplugin-provisioner" already exists
+Error from server (AlreadyExists): error when creating "STDIN": deployments.apps "csi-rbdplugin-provisioner" already exists.
+*/
+func getStdErr(errString string) string {
+	stdErrStr := "stderr:\n"
+	errStr := "error:\n"
+	stdErrPosition := strings.Index(errString, stdErrStr)
+	if stdErrPosition == -1 {
+		return ""
+	}
+	errPosition := strings.Index(errString, errStr)
+	if errPosition == -1 {
+		return ""
```

Comment:
Maybe return the remaining string after `stderr:`? Otherwise it may return an empty string, even when stderr was found.
Not a problem now, but it is possible that the output of the kubectl API changes in the future.

Reply:
Yes correct but not sure How it's going to look like also, if something changes will take care of it in the future.